splunk/splunk.go

```diff
@@ -17,3 +21,19 @@ func GetSplunkAppAPIEndpoint(host, port, ownerName, appName string) string {
 func GetSearchAPIEndpoint(appAPIEndpoint string) string {
 	return fmt.Sprintf("%s/search/v2/jobs/export", appAPIEndpoint)
 }
+
+//go:embed splunk_container_template.tmpl
+var splunkServiceManifest string
+
+const NotUpErrorMsg = "The %s instance is not up, cannot verify indexing for tests."
+
+// WithSplunkContainer builds a splunk container
+func WithSplunkContainer(t *testing.T, testFunc func(containerfixture.FixtureInfo)) {
```

Comment:
This is a test fixture, why move it into the Splunk module?

Reply:
Because i'm using it also in the fetch-uj-records_test.go and I couldn't import it if it wasn't in the module